device.yaml

```
@@ -1118,6 +1118,142 @@ PdStatus:
         ExitUsb4FollowingDrSwap: 0x4
         Reserved: catch_all
 
+DpConfig:
+  type: register
+  address: 0x51
+  access: RW
+  size_bits: 80
+  reset_value: 0x0000800000011C03
+  description: Display Port Configuration
+  fields:
+    EnableDpSvid:
+      base: bool
+      start: 0
+      description: Assert this bit to enable DisplayPort SVID.
+    DpMode:
+      base: bool
+      start: 1
+      description: Assert this bit to enable DisplayPort Alternate mode.
+    DpTransportSignalling:
+      base: uint
+      start: 10
+      end: 14
+      description: Signaling for transport of DisplayPort protocol.
+      conversion:
+        name: DpTransportSignalling
+        UsbSignaling: 0x0
+        DpSignaling: 0x1
+        Reserved: catch_all
+    DpPortCapability:
+      base: uint
+      start: 8
+      end: 10
+      description: Display port capabilities
+      conversion:
+        name: DpPortCapability
+        Reserved: 0x0
+        UfpDCapable: 0x1
+        DfpDCapable: 0x2
+        Reserved2: 0x3
+    DfpdPinAssignment:
+      base: uint
+      start: 16
+      end: 24
+      description: DFP_D Pin Assignments Supported. Each bit corresponds to an allowed pin assignment. Multiple pin assignments may be allowed.
+    UfpdPinAssignment:
+      base: uint
+      start: 24
+      end: 32
+      description: UFP_D Pin Assignments Supported. Each bit corresponds to an allowed pin assignment. Multiple pin assignments may be allowed.
+    MultiFunctionPreferred:
+      base: bool
+      start: 32
+      description: Assert this bit if multi-function is preferred.
+    DfpdUfpdConnected:
+      base: uint
+      start: 35
+      end: 37
+      description: This field indicates the status of the connection.
+      conversion:
+        name: DfpdUfpdConnected
+        NeitherConnected: 0x0
+        DfpDConnected: 0x1
+        UfpDConnected: 0x2
+        BothConnected: 0x3
```

Echoes
```suggestion
        Neither: 0x0
        DfpD: 0x1
        UfpD: 0x2
        Both: 0x3
```

Though this looks like a bitfield. I don't think device_driver can represent it that way though.

---

Echoes are fixed. The current implementation is using those enum values which seems to make sense (because we just have two things we just have "both" which is both of them.)

Anything else I should fix here?